packages/core/src/storage/vectorStore/PGVectorStore.ts

```diff
@@ -82,7 +82,9 @@ export class PGVectorStore implements VectorStore {
   private async getDb(): Promise<pg.Client> {
     if (!this.db) {
       try {
-        const { Client } = await import("pg");
+        const pg = await import('pg');
+        const { Client } = pg.default ? pg.default : pg;
```

Comment:
I can confirm this when trying locally, it's due to the use of `await import` so it can be supported in both ESM/non-ESM 

```suggestion
        const { Client } = pg.default || pg;
```

Reply:
Just needs that prettier step to merge